gpflow/conditionals/util.py

```diff
@@ -7,7 +7,7 @@
 def base_conditional(Kmn: tf.Tensor,
                      Kmm: tf.Tensor,
                      Knn: tf.Tensor,
-                     function: tf.Tensor,
+                     f: tf.Tensor,
```

Comment:
I agree that `function` name might not be appropriate. However, `f` is clearly not self-explanatory and depends a lot on the context. Moreover, we don't follow pep8 in naming, but still, we need to stick to those rules as far as it is possible.

Reply:
Happy to discuss name changes (I also find the module name unhelpful) - this was meant to bring it in line with the other functions in this file at least.

Reply:
agree we should move away from `f` . can I suggest `f_mu` and `q_sqrt` being `f_sqrt`. In general `f` looks like it s either, a `q_mu` or apparently just a set of points (dirac points, when `q_sqrt` is None). Another possibility is `q_mu`.

Reply:
how about f_mean and f_cov_chol ?

Reply:
I'm in favour of keeping `f` and `q_sqrt` for *now*. I agree that they are not the most appropriate names, but we should not ignore the cost of changing them to something else sub-optimal. I would rather spend time bundling the `f` and `q_sqrt` params in an appropriate class.

Reply:
is this related to the PR title?

Reply:
My opinion is that if you choose to rename `q_sqrt` globally in the repo then feel free to change it as well here. Now given the fact that you won't do that since I don't see many good reasons for it, and we keep discussing about having a distribution object for the posterior instead of `q_mu` and `q_sqrt` (this is the only good reason which makes a lot of sense but it is not trivial) I would suggest to keep it as is. Same goes for `f`